taxcalc/functions.py

```diff
@@ -405,10 +412,9 @@ def AMED(_fica, e00200, MARS, AMED_thd, _sey, AMED_trt,
     """
 
     # ratio of income subject to AMED tax = (1 - 0.5*(FICA_mc_trt+FICA_ss_trt)
-    _amed = AMED_trt * max(0, e00200 +
-                           max(0, _sey) * (1 - 0.5 *
-                                           (FICA_mc_trt + FICA_ss_trt)) -
-                           AMED_thd[MARS - 1])
+    _amed = AMED_trt * (max(0, e00200 - AMED_thd[MARS - 1]) + max(0, max(0,
```

Comment:
On Thu, 3 Dec 2015, Sean.Wang wrote:

> 
> In taxcalc/functions.py:
> 
> > @@ -405,10 +412,9 @@ def AMED(_fica, e00200, MARS, AMED_thd, _sey, AMED_trt,
> >      """
> > 
> >      # ratio of income subject to AMED tax = (1 - 0.5*(FICA_mc_trt+FICA_ss_trt)
> > -    _amed = AMED_trt * max(0, e00200 +
> > -                           max(0, _sey) * (1 - 0.5 *
> > -                                           (FICA_mc_trt + FICA_ss_trt)) -
> > -                           AMED_thd[MARS - 1])
> > +    _amed = AMED_trt * (max(0, e00200 - AMED_thd[MARS - 1]) + max(0, max(0,

Perhaps I don't understand this - but where does the .5 come from? Looking 
at the form, I don't see it anywhere - just a tax of .9% on wage income 
and self-employment income over a threshold. The income items aren't 
e00200 and e00900 - are you saying that the E-values used for the medicare 
tax differ from e00200 and e00900 by that amount?

dan